client/src/composables/config.ts

```diff
@@ -0,0 +1,20 @@
+import { computed, onMounted } from "vue";
+
+import { useConfigStore } from "@/stores/configurationStore";
+
+/* composable config wrapper */
+export function useConfig(fetchOnce = false) {
+    const store = useConfigStore();
+
+    const config = computed(() => store.config);
+    const isLoaded = computed(() => store.isLoaded);
```

Comment:
Maybe we should rename this to `isConfigLoaded` here?

Reply:
I wanted to avoid touching too many files to prevent imminent conflicts, but I really do prefer the explicit `isConfigLoaded`, all I needed was this little push :stuck_out_tongue_winking_eye: :+1: